proto/cosmwasm/wasm/v1/query.proto

```diff
@@ -317,6 +324,15 @@ message QueryContractsByCreatorResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+// QueryWasmLimitsConfigRequest is the request type for the
+// Query/WasmLimitsConfig RPC method.
+message QueryWasmLimitsConfigRequest {}
+
+// QueryWasmLimitsConfigResponse is the response type for the
+// Query/WasmLimitsConfig RPC method. It contains the JSON encoded limits for
+// static validation of Wasm files.
+message QueryWasmLimitsConfigResponse { string config = 1; }
```

Comment:
Do we have a specific struct that we can define for config or we want to keep it flexible?
In that case, we could evaluate using bytes instead of string.  WDYT?

Reply:
As mentioned in our chat, I want to keep it flexible. The reason for string instead of bytes is that I want to make it easy for contract devs to copy and paste the limits. They need pass them into cosmwasm-check. See here: https://github.com/CosmWasm/cosmwasm/blob/cf413c5ad6a58a87e0be894584f01506f3b2e0af/packages/check/src/main.rs#L41-L50